src/tools/tests/goldens/kotlin-schema-generation.cgtest

```diff
@@ -46,20 +40,23 @@ Schema(
     "accd28212161fc896d658e8c22c06051d1239e18",
     refinement = { _ -> true },
```

Comment:
It's cool to see the diff of a ts file map on to a cgtest file.

Reply:
Yeah, creepy ;)